src/api.ts

```diff
@@ -1,13 +1,13 @@
 import { MovieData } from './types/MovieData';
 import { ResponseError } from './types/ReponseError';
 
-const API_URL = 'https://www.omdbapi.com/?apikey=your-key';
+const API_URL = 'http://www.omdbapi.com/?i=tt3896198&apikey=b3f6f4e3';
```

Comment:
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/91411692/8b4fa648-fb2c-468b-a8d9-3873c2a6a842)
```suggestion
const API_URL = 'https://www.omdbapi.com/?i=tt3896198&apikey=b3f6f4e3';
```